test/custom_appbar_test.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:movies_widgets/movies_project_widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:network_image_mock/network_image_mock.dart';
+
+void main() {
+  Widget _buildWidget() {
+    return MaterialApp(
+      home: Scaffold(
+        body: MovieDetailsImage(
+          movieImage:
+              'https://www.themoviedb.org/t/p/w1280/qa6HCwP4Z15l3hpsASz3auugEW6.jpg',
+        ),
+      ),
+    );
+  }
```

Comment:
you could abstract this method to another file, and pass the widget to test via parameter

Reply:
Done , also corrected line 10 where i was passing the incorrect widget.